runbooks/data/deprecation-stored-files.md

```diff
@@ -8,13 +8,13 @@ Occasionally, we want to assess our Google Cloud Storage buckets for outdatednes
         * Note: [External tables](https://cloud.google.com/bigquery/docs/external-tables) in BigQuery, created from GCS objects via [our `create_external_tables` DAG](https://o1d2fa0877cf3fb10p-tp.appspot.com/dags/create_external_tables/grid) in Airflow, do not produce read or write data that shows up in the GCS request count metric we used in step one. If you find a reference to a deprecation candidate bucket within the [`create_external_tables` subfolder](https://github.com/cal-itp/data-infra/tree/main/airflow/dags/create_external_tables) of the data-infra repository, you should check [BigQuery audit logs](https://cloud.google.com/bigquery/docs/reference/auditlogs/#data_access_data_access) to see whether people are querying the external tables that rely on the deprecation candidate bucket (and if so, eliminate it from the deprecation list).
     * Post in `#data-warehouse-devs` and any other relevant channels in Slack (this may vary by domain; for example, if investigating a bucket related to GTFS quality, you may post in `#gtfs-quality`). Ask whether anybody knows of ongoing use of the bucket(s) in question. If there are identifiable stakeholders who aren't active in Slack, like external research partners, reach out to them directly.
 
-4. For each bucket that hasn't been removed from the deprecation list via the investigation in the last step, create a new bucket named "[EXISTING BUCKET NAME]-deprecated" with the default storage class set to Archive (setting the storage class is one of the bucket creation steps) and follow [these steps](https://cloud.google.com/storage/docs/moving-buckets#permissions-console) to transfer the original bucket contents into the newly created bucket(s). Delete the original bucket(s), inform stakeholders about the newly deprecated buckets via `#data-warehouse-devs` and other relevant channels, and monitor for two weeks for any new code or process breakages related to the deletion of the old buckets.
+4. For each bucket that hasn't been removed from the deprecation list via the investigation in the last step, tag the bucket as "deprecated" and remove access for any relevant automated users or user groups (we want to intentionally break automated access so that errors occur if the bucket's data was still being referenced undetected). Inform stakeholders about the newly deprecated buckets via `#data-warehouse-devs` and other relevant channels, and monitor for two weeks for any new code or process breakages related to the changed bucket permissions.
```

Comment:
Don't we still want to change the data class to archive/cold storage? I thought the goal is to reduce the billing from keeping this data even if we don't delete it. Also can/should there be a link to how to add a label to a bucket in GCS?

Reply:
I'm coming back to the storage class guidance later, since the mechanisms for changing objects in place are more advanced than the previous guidance. It's still in the cards, just not part of this PR.

Reply:
Ok. I think if many of the very core steps are still undecided, may be clearer to just leave this as draft and only merge once those pieces are known (keeping a docs PR in draft for some period of time less risky than code changes). IMO It's confusing to merge stuff this incrementally where we know this doesn't actually reflect a desired process state.

Reply:
The language has now been updated, and Andrew and I got on the same page about process [here](https://jarvus.slack.com/archives/C04A23BSMML/p1692391198311689) while discussing what to do with the partially complete Round 1 deprecations. Ready for a second look.